tests/kernel/threads/thread_apis/prj.conf

```diff
@@ -1,4 +1,6 @@
 CONFIG_ZTEST=y
 CONFIG_THREAD_MONITOR=y
-CONFIG_HEAP_MEM_POOL_SIZE=256
 CONFIG_THREAD_CUSTOM_DATA=y
+CONFIG_INIT_STACKS=y
```

Comment:
why is CONFIG_INIT_STACKS enabled? does the test use it?

Reply:
we don't need this in this test. I will remove.